crates/re_viewer/src/ui/top_panel.rs

```diff
@@ -153,6 +157,35 @@ fn top_bar_ui(
     });
 }
 
+/// Shows clickable website link as an image (text doesn't look as nice)
+fn website_link_ui(ui: &mut egui::Ui, app: &mut App) {
+    let icon_image = app.re_ui().icon_image(&re_ui::icons::RERUN_IO_TEXT);
+
+    let desired_height = ui.max_rect().height();
+    let desired_height = desired_height.at_most(28.0); // figma size 2023-02-03
+
+    let image_size = icon_image.size_vec2() * (desired_height / icon_image.size_vec2().y);
+    let texture_id = icon_image.texture_id(ui.ctx());
+    let response = ui.add(egui::ImageButton::new(texture_id, image_size));
```

Comment:
egui uses the pointing hand cursor for links; it lets native users know that the click is going to open a link

```suggestion
    let response =
        ui.add(egui::ImageButton::new(texture_id, image_size))
        .on_hover_cursor(CursorIcon::PointingHand);
```